src/index.js

```diff
@@ -1,59 +1,63 @@
 import './style.css';
-import completed from './completedTasks.js';
-import * as LS from './localStorage.js';
-
-const todoList = [
-  {
-    description: 'Task 1',
-    completed: false,
-    index: 1,
-  },
-  {
-    description: 'Task 3',
-    completed: true,
-    index: 3,
-  },
-  {
-    description: 'Task 4',
-    completed: false,
-    index: 4,
-  },
-  {
-    description: 'Task 2',
-    completed: true,
-    index: 2,
-  },
-];
-
-if (!JSON.parse(localStorage.getItem('todo-list'))) {
-  LS.storeList(LS.setSortingIndex(todoList));
-}
+import LSOperations from './localStorageOperations.js';
+import manipulateHtml from './manipulateHtml.js';
 
-const btn = document.querySelector('.btn');
-const list = document.querySelector('#todo-list');
-function displayList() {
-  const LocalStoragelist = LS.getList();
-  LocalStoragelist.forEach((item) => {
-    const newItem = document.createElement('LI');
-    const newIcon = document.createElement('I');
-    newIcon.classList.add('fa-solid', 'fa-ellipsis-vertical');
-    const checkbox = document.createElement('input');
-    checkbox.type = 'checkbox';
-    checkbox.checked = item.completed;
-    checkbox.id = item.index;
-    checkbox.classList.add('checkbox');
-    const label = document.createElement('label');
-    const textnode = document.createTextNode(item.description);
-    label.appendChild(textnode);
-    newItem.appendChild(checkbox);
-    newItem.appendChild(label);
-    newItem.appendChild(newIcon);
-    list.insertBefore(newItem, btn);
+const todoList = [];
+
+const renderUI = () => {
+  if (!JSON.parse(localStorage.getItem('todo-list'))) {
+    LSOperations.storeList(LSOperations.setSortingIndex(todoList));
+  }
+
+  manipulateHtml.displayList();
+
+  const checkboxes = document.querySelectorAll('.checkbox');
+  [...checkboxes].forEach((button) => {
+    button.addEventListener('change', LSOperations.completeTask);
+  });
+
+  const inputs = document.querySelectorAll('.description');
+  [...inputs].forEach((input) => {
+    input.addEventListener('focusout', LSOperations.updateTask);
+  });
+
+  [...inputs].forEach((input) => {
+    input.addEventListener('focus', (event) => {
+      event.target.style.backgroundColor = '#fff4bf';
+      event.target.parentElement.style.backgroundColor = '#fff4bf';
+      event.target.nextSibling.classList.add('display-none');
+      event.target.nextSibling.nextSibling.classList.remove('display-none');
+    });
+  });
+
+  const trashes = document.querySelectorAll('.fa-trash');
+  [...trashes].forEach((trash) => {
+    trash.addEventListener('click', (event) => {
+      LSOperations.remove(event);
+      const oldList = document.querySelectorAll('.todoItem');
+      [...oldList].forEach((e) => e.remove());
+      renderUI();
+    });
```

Comment:
- [ ] great job on implementing the functionality 👍  however the remove function is not working when clicked. please make sure it works before submitting another code review request.

Reply:
Done